gobblin-utility/src/test/resources/templates/wikiSample.template

```diff
@@ -0,0 +1,23 @@
+job.name=PullFromWikipedia
```

Comment:
It would also be nice to have this as a non-test sample template, usually people start with wikipedia example. Have the page titles as the only mandatory attribute.